### PR TITLE
Create helpdesk.md

### PR DESCRIPTION
**Pull request by danielsischy**

### CODE REVIEW

File: operations/IT/helpdesk.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# Helpdesk
+
+The Mattermost IT team provides technical support to all Mattermost staff members and is dedicated to servicing the technology support needs of all staff by providing detailed resolutions and general system information for common problems.
+
+Through the internal helpdesk solution, staff members are able to request application access, elevated permissions, assistance troubleshooting software/hardware issues, flag security issues, and ask for help with integrations, as well as submit general inquiries.
```
Comment (justinegeffen) [inline operations/IT/helpdesk.md]:
```suggestion
Through the internal helpdesk solution, staff members are able to request application access, elevated permissions, assistance with troubleshooting software/hardware issues, flag security issues, and ask for help with integrations, as well as submit general inquiries.
```

File: operations/IT/helpdesk.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+Through the internal helpdesk solution, staff members are able to request application access, elevated permissions, assistance with troubleshooting software/hardware issues, flag security issues, and ask for help with integrations, as well as submit general inquiries.
+
+## Availability
+
+Helpdesk services are available during the following hours of operation, except during office closures and company observed holidays.
+
+* Monday through Friday **9:00 a.m. - 5:00 p.m. EST**
+
+Support is available outside of working hours in the case of an emergency. Any issue classified as **"urgent"** is considered an emergency. **"high"** level incidents may also fall under scope as an emergency depending on the request.
```
Comment (justinegeffen) [inline operations/IT/helpdesk.md]:
```suggestion
Support is available outside of working hours in the case of an emergency. Any issue classified as **"urgent"** is considered an emergency. Incident classified as **"high"** level may also fall under the scope of an emergency depending on the request.
```

Comment (danielsischy) [reply operations/IT/helpdesk.md]: Hey @justinegeffen should this be "Incidents" vs "Incident"?

File: operations/IT/helpdesk.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+* A clear and specific description of the problem or request, including information regarding any error messages you may have received.
+
+## Service scope
+
+All requests considered an **Incident** or **Service Request** are covered under the service scope.
+
+* An **Incident** is considered 'an unplanned interruption to an IT service or reduction in the quality of an IT service'.
+* A **Service Request** is considered 'a request for something to be provided - for example, a request for information, access to a platform, or change within an existing platform'.
+
+## Questions
```
Comment (TQuock) [inline operations/IT/helpdesk.md:54]: Is there an IT Channel that you also want to add for staff to ping on?

**Review by TQuock — COMMENTED** [review]
Overall, looks great! Had one comment about adding a MM channel that others can reach out to rather than email as there will be a thread for other staff members to be able to reference so you don't need to repeat yourself as much.

File: operations/IT/helpdesk.md (hunk 3 above)
Comment (danielsischy) [reply operations/IT/helpdesk.md:54]: That makes sense! I added it to the document. Thanks for the feedback Tim

File: operations/IT/helpdesk.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+If you have any questions or feedback, please reach out to the IT team at IT@mattermost.com or join the IT channel in Mattermost https://community-daily.mattermost.com/private-core/channels/it
```
Comment (justinegeffen) [inline operations/IT/helpdesk.md]:
```suggestion
If you have any questions or feedback, please reach out to the IT team at IT@mattermost.com or join the [IT channel in Mattermost](https://community-daily.mattermost.com/private-core/channels/it).
```

**Review by justinegeffen — DISMISSED** [review]

**Comment by justinegeffen** [comment]
I've added this to the TOC, @danielsischy! Please let me know if you'd like it in a different place. :)

**Comment by danielsischy** [comment]
Hey @justinegeffen I was hoping to create a top level IT folder under Operations where I could start to populate IT articles. If it can't live under top level Operations, then creating a IT folder under "Company Operations" where this could live and I could start populating more articles would be fine as well. Thanks!

**Comment by danielsischy** [comment]
Hey @justinegeffen I was hoping to have this article under its own department (IT) on the top level of operations like "Finance", "Sales", etc. so I could start populating the IT folder with other articles. If that's not possible, then creating an IT folder for the helpdesk article to live under "computer operations" would be fine. Thanks!

**Comment by justinegeffen** [comment]
Thanks, @danielsischy! That makes sense, let me sort that out quick.

**Comment by justinegeffen** [comment]
I've added the entry to the TOC, @danielsischy. Feel free to move it around as needed! :) I also added a readme.md as a landing page.

**Review by DSchalla — APPROVED** [review]

**Review by justinegeffen — APPROVED** [review]

**Review by TQuock — APPROVED** [review]

**Pull request merged**